app/org/sagebionetworks/bridge/services/StudyService.java

```diff
@@ -192,6 +198,33 @@ public Study createStudy(Study study) {
         return study;
     }
 
+    public String createSynapseAccount(SynapseAccount synapseAccount) throws SynapseException {
+        checkNotNull(synapseAccount, Validate.CANNOT_BE_NULL, "synapse account");
+        if (!synapseAccount.isValid()) {
+            throw new BadRequestException("Account information is not sufficient.");
+        }
+
+        NewUser newUser = new NewUser();
+        newUser.setEmail(synapseAccount.getEmail());
+        newUser.setFirstName(synapseAccount.getFirstName());
+        newUser.setLastName(synapseAccount.getLastName());
+        newUser.setUserName(synapseAccount.getUsername());
+
+        // if the validation failed, throw bad request exception
+        try {
+            synapseClient.createUser(newUser);
+        } catch (SynapseServerException e) {
+            throw new BadRequestException(e.getMessage());
```

Comment:
Should we add a unit test to verify error handling?

Also, how do we distinguish between a bad request and (for example) SynapseUnavailableException?